client/blocks/get-apps/mobile-download-card.jsx

```diff
@@ -188,45 +184,20 @@ class MobileDownloadCard extends Component {
 				</div>
 
 				{ featureIsEnabled && (
-					<div className="get-apps__sms-subpanel">
+					<div className="get-apps__qr-code-subpanel">
 						<p>
 							<strong>{ translate( 'Ready to WordPress on the go?' ) }</strong>
 							<br />
-							{ translate(
-								'We’ll send you an SMS message with a download link for the right app for your mobile device.'
-							) }
+							{ translate( 'Scan the code using your mobile phone to download the app.' ) }
 						</p>
 
-						<div className="get-apps__sms-field-wrapper">
-							<QuerySmsCountries />
-
-							{ hasAllData ? (
-								<FormPhoneInput
-									countriesList={ this.props.countriesList }
-									initialCountryCode={ countryCode }
-									initialPhoneNumber={ number }
-									phoneInputProps={ {
-										onKeyUp: this.onKeyUp,
-									} }
-									onChange={ this.onChange }
-								/>
-							) : (
-								<>
-									<FormPhoneInput countriesList={ this.props.countriesList } isDisabled={ true } />
-								</>
-							) }
-						</div>
-						<div className="get-apps__sms-button-wrapper">
-							<Button
-								className="get-apps__sms-button"
-								onClick={ this.onSubmit }
```

Comment:
We can probably also delete some of these `onSubmit, onKeyUp, onChange` actions that are found in this deleted code.

Reply:
Yes, agreed – I took a shot at it before and ended up breaking things. If I super-promise to come back to this, could we land this now and land a separate cleanup PR next?

Reply:
Sure.